src/__tests__/InputForm.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import InputForm from '../components/InputForm';
+
+describe('<InputForm />', () => {
+  const changeInput = () => {};
+  const fetchLocation = () => {};
+  const changeStatus = () => {};
+
+  it('disables submit button with no input text', () => {
+    const wrapper = shallow(
+      <InputForm
+        inputText={''}
+        changeInput={changeInput}
+        fetchLocation={fetchLocation}
+        changeStatus={changeStatus}
+      />,
+    );
+    const button = wrapper.find('input[type="submit"]');
+    expect(button.props().disabled).toEqual(true);
```

Comment:
you're killin' me with this `.toEqual(true)` business. Rock a .toBe(true) or even better expect props toHaveProperty. use your matchers! https://facebook.github.io/jest/docs/expect.html#tohavepropertykeypath-value